UnitsNet/UnitConverter.cs

```diff
@@ -133,11 +131,20 @@ public bool TryGetConversionFunction(Type fromType, Enum from, Type toType, Enum
 
         public bool TryGetConversionFunction(ConversionFunctionLookup conversionLookup, out ConversionFunction conversionFunction)
         {
-            return conversionFunctions.TryGetValue(conversionLookup, out conversionFunction);
+            return _conversionFunctions.TryGetValue(conversionLookup, out conversionFunction);
         }
 
 #endif
 
+        private static readonly Type[] QuantityTypes = UnitsNetAssembly.GetTypes()
+            .Where(typeof(IQuantity).IsAssignableFrom)
+            .Where(x => x.IsClass() || x.IsValueType()) // Future-proofing: we are discussing changing quantities from struct to class
+            .ToArray();
+
+        private static readonly Type[] UnitTypes = UnitsNetAssembly.GetTypes()
```

Comment:
There is some overlap here and in my PR, we should later merge so we have a single way to enumerate all unit types and quantity types.